packages/block-editor/src/components/copy-handler/index.js

```diff
@@ -130,6 +130,14 @@ export function useClipboardHandler() {
 			if ( event.type === 'cut' ) {
 				removeBlocks( selectedBlockClientIds );
 			} else if ( event.type === 'paste' ) {
+				if (
+					event.target.classList.contains(
+						'block-editor-rich-text__editable'
```

Comment:
Can't we check [`event.defaultPrevented`](https://developer.mozilla.org/en-US/docs/Web/API/Event/defaultPrevented)?

Reply:
We use a similar technique in other places. It also makes it a bit more general and not rich text specific. Other inner components could handle paste as well.

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/34430/commits/d0d16cfa8fc8eca94efd4d79adb5f1059dbf18a1 I had inconsistent results earlier, but missed that we were calling it above in this same handler.